modules/gapi/misc/python/test/test_gapi_core.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import numpy as np
+import cv2 as cv
+
+from tests_common import NewOpenCVTests
+
+
+# Plaidml is an optional backend
+pkgs = [
+         cv.gapi.core.ocl.kernels(),
+         cv.gapi.core.cpu.kernels(),
+         cv.gapi.core.fluid.kernels()
+         # cv.gapi.core.plaidml.kernels()
+       ]
+
+
+class gapi_core_test(NewOpenCVTests):
+
+    def test_add(self):
+        # TODO: Extend to use any type and size here
+        sz = (1280, 720)
+        in1 = np.random.randint(0, 100, sz).astype(np.uint8)
+        in2 = np.random.randint(0, 100, sz).astype(np.uint8)
+
+        # OpenCV
+        expected = in1 + in2
+
+        # G-API
+        g_in1 = cv.GMat()
+        g_in2 = cv.GMat()
+        g_out = cv.gapi.add(g_in1, g_in2)
```

Comment:
I think that it's better to show in tests the best codestyle to work with G-API in Python. So maybe we need `from cv2 import gapi` and use as
```python
g_out = gapi.add(g_in1, g_in2)
```

Reply:
It's better to leave `cv.gapi.add` because it's a little bit strange when it use `cv.GMat` and `gapi.add`